src/cunit/patches/TestRun.patch

```diff
@@ -0,0 +1,69 @@
+--- CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c	2024-04-05 17:03:33.892651597 +0100
```

Comment:
ah this patch is the wrong way round, all the - should be + and all the + should be minus..

So your left & right are in the wrong place I think.

Reply:
Done.